Library/ViewModels/PaymentMethodsViewModel.swift

```diff
@@ -67,9 +67,30 @@ PaymentMethodsViewModelInputs, PaymentMethodsViewModelOutputs {
       paymentMethodsValues.takeWhen(deletePaymentMethodEventsErrors)
     )
 
+    let cardsDeletedSinceReload = Signal.merge(
+      self.didDeleteCreditCardSignal.map { card -> GraphUserCreditCard.CreditCard in card },
```

Comment:
I had to help the compiler along with the types here, it wasn't able to infer that this can sometimes be a `raphUserCreditCard.CreditCard` and sometimes `nil`. We use `nil` just to reset the accumulator in the `scan`.